app/views/devise/registrations/_edit_profile.html.haml

```diff
@@ -25,7 +25,7 @@
           %br/
           To change your profile picture, visit
           = succeed "." do
-            = link_to 'gravatar.com', "http://gravatar.com/"
+            = link_to 'gravatar.com', "https://gravatar.com/"
```

Comment:
A handy trick we can do is use urls like "//gravtar.com"; and it will assume http or https depending on how the page is served. But this is great for now :)